lua/lib/populate.lua

```diff
@@ -6,7 +6,8 @@ local api = vim.api
 local luv = vim.loop
 
 local M = {
-  show_ignored = false
+  show_ignored = false,
+  show_hidden = vim.g.lua_tree_show_hidden == 1,
```

Comment:
i think this should be called `show_dotfiles` for clarity though.
the option should be `lua_tree_hide_dotfiles` because it makes more sense to show them by default.

Reply:
sure